spot-electron/src/logger/fileLogger.js

```diff
@@ -0,0 +1,43 @@
+const log = require('electron-log');
+const { app } = require('electron');
+const path = require('path');
+
+log.transports.ipc.level = false;
+log.transports.file.resolvePath = () => path.join(app.getPath('logs'), 'spot-logs.log');
+log.transports.file.format = '{iso} {text}';
+log.transports.file.maxSize = 1048576 * 20; // 20mb
+
+const JITSI_TO_ELECTRON_LEVELS = {
+    error: 'error',
+    warn: 'warn',
+    info: 'info',
+    trace: 'verbose',
+    debug: 'debug',
+    log: 'log'
+};
+
+const ELECTRON_LEVEL_INDEX_MAP = {
+    0: JITSI_TO_ELECTRON_LEVELS.debug,
+    1: JITSI_TO_ELECTRON_LEVELS.info,
+    2: JITSI_TO_ELECTRON_LEVELS.warn,
+    3: JITSI_TO_ELECTRON_LEVELS.error
+};
+
+/**
+ * Logs a message to file.
+ *
+ * @param {string} level - The log level index.
+ * @param {string} message - The main string to be logged a warning.
+ * @param {Object} [context] - Additional information to be logged.
+ * @returns {void}
+ */
+function logToFile(level, message, context) {
+    const logType = ELECTRON_LEVEL_INDEX_MAP[level];
+    const logMethod = log[logType] || log.log;
+
+    logMethod && logMethod(`[${logType}]`, message, context ? JSON.stringify(context) : '');
```

Comment:
Don't we want to format the context like in the other logger?

Reply:
Ahh, forgot to delete this parameter as well. In the event `console-message`we receive `message` (the full log text from console) thus we don't have anymore a `context` in this use case (https://www.electronjs.org/docs/latest/api/webview-tag#event-console-message)